rundeckapp/grails-app/conf/application.groovy

```diff
@@ -116,6 +113,7 @@ grails.plugin.springsecurity.interceptUrlMap = [
         [pattern: '/user/loggedout', access: ['permitAll']],
         [pattern: '/feed/**',        access: ['permitAll']],
         [pattern: '/api/**',         access: ['permitAll']],
+        [pattern: '/health',         access: ['permitAll']],
```

Comment:
this seems unrelated to typed configs, why was this added?

Reply:
This is added for the Pro side of this feature. When Rundeck registers itself with Consul, Consul likes to have a health check endpoint so it can test the apps that have registered and make sure they're still alive. The /heath endpoint is the standard endpoint it hits.